executors/node/collator.js

```diff
@@ -39,13 +44,24 @@ module.exports = {
         result_bool = false;
       }
       outputLine = {'label':json['label'],
-                    'result': result_bool,
-                    'compare_result': compared,
                    }
-
-      if (result != true) {
+      if (result == true) {
```

Comment:
Question about initializations of `result` and `result_bool`, even though they're not in this PR: shouldn't `result` and `result_bool` always be identical, by definition?

If so, can we just use one of them instead of both, when it comes to the code here in the body of this if statement?

Reply:
Yes, I'll remove result_bool.